table/block_based_table_builder.cc

```diff
@@ -43,6 +43,7 @@
 #include "util/compression.h"
 #include "util/crc32c.h"
 #include "util/memory_allocator.h"
+#include "util/random.h"
```

Comment:
It looks like this header inclusion can be removed.

Reply:
Fixed.